app/views/travel_sponsorships/index.html.haml

```diff
@@ -5,6 +5,7 @@
 .navbar
   .navbar-inner
     = search_form_for @q, :class => "navbar-form pull-right" do |f|
+      = f.text_field(:user_nickname_or_event_name_or_user_profile_full_name_cont, :placeholder => t(:requests_search_placeholder), :class=>"input-xlarge")
```

Comment:
I think i_cont is [not usable yet](https://github.com/activerecord-hackery/ransack/wiki/Basic-Searching#i_cont-work-in-progress-dont-use-yet). I tried putting it in there, but it gives an error of ``undefined method``.